docs/content/querying/groupbyquery.md

```diff
@@ -214,3 +214,4 @@ When using the "v2" strategy, the following query context parameters apply:
 |`maxMergingDictionarySize`|Can be used to lower the value of `druid.query.groupBy.maxMergingDictionarySize` for this query.|
 |`maxOnDiskStorage`|Can be used to lower the value of `druid.query.groupBy.maxOnDiskStorage` for this query.|
 |`sortByDimsFirst`|Sort the results first by dimension values and then by timestamp.|
+|`pushDownLimit`|Limits/orderbys are applied at the merge buffer level (only the top N results are kept in the merge buffers, reducing the workload at the broker level), when all fields in the orderby are part of the grouping key. When the sorting order uses fields that are not in the grouping key, this can result in approximate results with unknown accuracy, so this optimization is disabled by default in that case. Enabling this context flag turns on limit push down for limit/orderbys with non-grouping key columns.|
```

Comment:
i don't think user needs to know about the "merge buffer stuff"... it would suffice to say that...

broker node always pushes down limits to historical nodes, when it is guaranteed to not alter the overall results i.e.  when all fields in the orderby...



on a side note, similar to topN, we can improve accuracy of the aggregator case if we inflate the limit to say 100x than what user specified when pushing to historicals.